bundles/org.openhab.binding.mqtt.fpp/src/main/java/org/openhab/binding/mqtt/fpp/internal/handler/FPPPlayerHandler.java

```diff
@@ -0,0 +1,280 @@
+            }
+        }
+        if (channelId.equals(CHANNEL_TESTING)) {
+            if (command == OnOffType.OFF) {
+                executePost("/api/testmode", GetTestMode(0));
+            } else if (command == OnOffType.ON) {
+                executePost("/api/testmode", GetTestMode(1));
+            }
+        }
+    }
+
+    private String GetTestMode(int enable) {
+        return "{\"mode\":\"RGBChase\",\"subMode\":\"RGBChase-RGB\",\"cycleMS\":1000,\"colorPattern\":\"FF000000FF000000FF\",\"enabled\":"
+                + enable + ",\"channelSet\": \"1-1048576\",\"channelSetType\": \"channelRange\"}";
+    }
+
+    @Override
+    public void initialize() {
+        config = getConfigAs(ConfigOptions.class);
+
+        fullStatusTopic = config.playerMQTT + "/" + STATUS_TOPIC;
+        fullVersionTopic = config.playerMQTT + "/" + VERSION_TOPIC;
+        bridgeStatusChanged(getBridgeStatus());
+    }
+
+    @Override
+    public void processMessage(String topic, byte[] payload) {
+        String state = new String(payload, StandardCharsets.UTF_8);
+        logger.trace("Received the following new FPP state:{}:{}", topic, state);
+
+        if (topic.endsWith(STATUS_TOPIC)) {
+            processIncomingState(state);
+        } else if (topic.endsWith(VERSION_TOPIC)) {
+            updateState(CHANNEL_VERSION, new StringType(state));
+            updateStatus(ThingStatus.ONLINE);
+        }
+    }
+
+    public ThingStatusInfo getBridgeStatus() {
+        Bridge b = getBridge();
+        if (b != null) {
+            return b.getStatusInfo();
+        } else {
+            return new ThingStatusInfo(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE, null);
+        }
+    }
+
+    @Override
+    public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
+        if (bridgeStatusInfo.getStatus() == ThingStatus.OFFLINE) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+            connection = null;
+            return;
+        }
+        if (bridgeStatusInfo.getStatus() != ThingStatus.ONLINE) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
+            return;
+        }
+
+        Bridge localBridge = this.getBridge();
+        if (localBridge == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED,
+                    "Bridge is missing or offline, you need to setup a working MQTT broker first.");
+            return;
+        }
+        ThingHandler handler = localBridge.getHandler();
+        if (handler instanceof AbstractBrokerHandler abh) {
+            final MqttBrokerConnection connection;
+            try {
+                connection = abh.getConnectionAsync().get(500, TimeUnit.MILLISECONDS);
+            } catch (InterruptedException | ExecutionException | TimeoutException ignored) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED,
```

Comment:
Calling this method is initializing the connection (if not already) and returning it. The init envolves a tcp/udp connection to the broker inclukding possible auhtentication.
Long story short, i hope all systems respond in 500ms but i think this is a bit to tight. Maybe 5000 is better.